### ISSUE TITLE
Syntax highlighting fails to accept load-rdf source

### ISSUE THREAD

**Issue by mkroetzsch** [knowsys/rulewerk-eclipse#5]
The correct input
```
@source triple[3]: load-rdf("rdf.nt") .
```
is not accepted by the Eclipse plugin. An error appears for ```load-rdf```.

**Comment by alloka** [comment]
Dear markus,


I think you are having a problem from the previous version of the plugin, you should remove the whole file in the project and add it from the beginning, because for me this works normally as well..

________________________________

**Comment by mkroetzsch** [comment]
I am on the current master branch and I have used maven install both in Eclipse and in the command line. I will try if I can do some more to get the correct behaviour of the master ...

**Comment by mkroetzsch** [comment]
I have run "Maven clean" both from Eclipse and from the command line. After this, the project shows many errors and when I try to "Run as Eclipse Application", I get the error ```Plug-in rls4eclipse.ui was unable to load class rls4eclipse.ui.RLSExecutableExtensionFactory.``` So it seems that the application is really gone.

Then I have then used "Maven install" from Eclipse to rebuild the application. After this (and some Maven Project updating), the errors are gone and I can again "Run as Eclipse Application". However, the issue then re-appeared and looked as before.

I then deleted the whole project from disk and re-imported everything from github. After Maven install, the issue was finally gone. We should keep an eye on the build process to see if these problems occur in the future as well. Deleting the whole repository locally to get an update from master is not really a workable solution.

Closing this bug now since it does not occur in master anymore.
